### PR TITLE
docs: Add Docs for  Interruption Handling

### PR DESCRIPTION
**Pull request by jonathan-innis**
Fixes # 

**Description**

- Creates docs for enabling Interruption Handling

**How was this change tested?**

* N/A

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note
None
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/tasks/globalsettings.md [hunk 1]
```diff
@@ -62,6 +62,9 @@ data:
   # The VM memory overhead as a percent that will be subtracted 
   # from the total memory for all instance types 
   aws.vmMemoryOverheadPercent: "0.075"
+  # Interruption Handling is currently in BETA and is disabled by default. Enabling interruption handling may
```
Comment (ellistarn) [inline website/content/en/preview/tasks/globalsettings.md:65]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ are enabled in beta. I'd call this one alpha, tbh.

File: website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md [hunk 2]
```diff
@@ -130,13 +134,26 @@ The new stack has only one user, `admin`, and the password is stored in a secret
...
+By enabling Karpenter interruption handling, you are allowing Karpenter to provision the following infrastructure in your account:
+
+* SQS Queue (named after your EKS cluster name)
+* EventBridge Rules (prefixed with `Karpenter-`)
+
+{{% alert title="Note" color="primary" %}}
+All infrastructure that Karpenter creates and deploys to your account is tagged with the `karpenter.sh/discovery: ${CLUSTER_NAME}` and the `karpenter.sh/managed-by: ${CLUSTER_NAME}` tags for ease of discovery
+{{% /alert %}}
+
+Permission to provisions these rules is given through the `KarpenterEventPolicy` provisioned through the [CloudFormation template](#create-the-karpenternode-iam-role) deployed above.
```
Comment (bwagner5) [inline website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md:150]:
```suggestion
Permission to provision these rules is given through the `KarpenterEventPolicy` created through the [CloudFormation template](#create-the-karpenternode-iam-role) deployed above.
```

File: website/content/en/preview/concepts/_index.md [hunk 3]
```diff
@@ -116,7 +117,18 @@ To perform these actions, Karpenter simulates all pods being evicted from a cand
...
+### Interruption
+
+If interruption-handling is enabled for the controller, Karpenter will watch for upcoming involuntary interruption events that would cause disruption to your workloads. These interruption events include:
+
+* Spot Interruption Warnings
+* Scheduled Change Health Events (Maintenance Events)
+* Instance Terminating Events
+* Instance Stopping Events
+
+When Karpenter detects one of these events will occur to your nodes, it automatically cordons, drains, and terminates the node(s) ahead of the interruption event to give the maximum amount of time for workload cleanup prior to compute disruption. This enables scenarios where the `terminationGracePeriod` for your workloads may be long or cleanup for your workloads is critical, and you want enough time to be able to gracefully clean-up your pods.
```
Comment (bwagner5) [inline website/content/en/preview/concepts/_index.md:131]: can you make a note that we don't support Rebalance Recommendations today so users will need to continue to use NTH if they want that functionality?

**Review by bwagner5 — APPROVED** [review]
LGTM

**Pull request merged**